app/views/katello/api/v2/repositories/base.json.rabl

```diff
@@ -0,0 +1,39 @@
+object @resource
+
+extends 'katello/api/v2/common/identifier'
+extends 'katello/api/v2/common/syncable'
+
+attributes :content_type
+attributes :url,
+           :relative_path
```

Comment:
Mind moving this to a single line? Don't know anywhere else that we multiline this type of declaration.